### ISSUE TITLE
Is the test set going to be released to public

### ISSUE THREAD

**Issue by wanglamao** [drivendataorg/tissuenet-cervical-biopsies#1]
Hi! 

I would like to know wether the test set is going to be released to public. 

Thank you!

**Comment by geektortoise** [comment]
@chlmarchal

**Comment by geektortoise** [comment]
Hello, 

We are interested by the test set but also the training set as 3rd and 4rth algo don't contain the trained model.

Have a nice day

**Comment by geektortoise** [comment]
@wanglamao , I got the information that the publication of the sets is still evaluated by the CNIL (National French Data Protection Officer)
